__tests__/core/shared/event-emitter.tests.ts

```diff
@@ -101,7 +101,7 @@ test("should emit correctly a test result event with test decorations", () => {
       test: testName,
       state: TestState.Passed,
       message: "Expected 'test-project' to equal 'test-projectsdsdsd'. (line:22 column:23)",
```

Comment:
Could you update this message with the right line too?

Reply:
The message is correct because the change I made only involves the position of the decorator.
<img width="905" alt="Schermafbeelding 2021-08-28 om 22 43 17" src="https://user-images.githubusercontent.com/7029563/131230580-6ed352db-42a8-40f7-a7f6-9d7005ca1557.png">
In the example test I created, you can see that the message is correct.

Reply:
Oh right, good call.